<launch>

  	<node pkg="map_compression" type="selectBySaliency" name="selectBySaliency" output="screen" >

	<!--param name="loadMapName" value="/home/yh/mapModel/2019/07.11/loc/loc_kitti_saliency.ply"/>
	<param name="isSave" value="true"/>
  	<param name="saveMapName" value="/home/yh/mapModel/2019/07.11/loc/loc_kitti_b1000_saliency.ply"/-->

	<param name="loadMapName" value="/home/yh/mapModel/2019/07.12/compare/wholeMap_yq_saliency_loc.ply"/>
	<param name="isSave" value="true"/>
  	<param name="saveMapName" value="/home/yh/mapModel/2019/07.12/loc/loc_yq_b1000_saliency.ply"/>

	<param name="saliencyThreshold" value="1.355"/>

  </node>
</launch>
